pysph/examples/rigid_body/sphere_in_vessel_akinci.py

```diff
@@ -0,0 +1,220 @@
+"""A sphere of density 500 falling into a hydrostatic tank (15 minutes)
+
+Check basic equations of SPH to throw a ball inside the vessel
+"""
+from __future__ import print_function
+import numpy as np
+import matplotlib.pyplot as plt
+
+# PySPH base and carray imports
+from pysph.base.utils import (get_particle_array_wcsph,
+                              get_particle_array_rigid_body)
+from pysph.base.kernels import CubicSpline
+
+from pysph.solver.solver import Solver
+from pysph.sph.integrator import EPECIntegrator
+from pysph.sph.integrator_step import WCSPHStep
+
+from pysph.sph.equation import Group
+from pysph.sph.basic_equations import (XSPHCorrection, ContinuityEquation,
+                                       SummationDensity)
+from pysph.sph.wc.basic import TaitEOSHGCorrection, MomentumEquation
+from pysph.solver.application import Application
+from pysph.sph.rigid_body import (
+    BodyForce, SummationDensityBoundary, CutoffDensity, RigidBodyCollision,
+    LiuFluidForce, RigidBodyMoments, RigidBodyMotion, AkinciRigidFluidCoupling,
+    RK2StepRigidBody)
+
+
+def create_boundary():
+    dx = 2
+
+    # bottom particles in tank
+    xb = np.arange(-2 * dx, 100 + 2 * dx, dx)
+    yb = np.arange(-2 * dx, 0, dx)
+    xb, yb = np.meshgrid(xb, yb)
+    xb = xb.ravel()
+    yb = yb.ravel()
+
+    xl = np.arange(-2 * dx, 0, dx)
+    yl = np.arange(0, 250, dx)
+    xl, yl = np.meshgrid(xl, yl)
+    xl = xl.ravel()
+    yl = yl.ravel()
+
+    xr = np.arange(100, 100 + 2 * dx, dx)
+    yr = np.arange(0, 250, dx)
+    xr, yr = np.meshgrid(xr, yr)
+    xr = xr.ravel()
+    yr = yr.ravel()
+
+    x = np.concatenate([xl, xb, xr])
+    y = np.concatenate([yl, yb, yr])
+
+    return x * 1e-3, y * 1e-3
+
+
+def create_fluid():
+    dx = 2
+    xf = np.arange(0, 100, dx)
+    yf = np.arange(0, 150, dx)
+    xf, yf = np.meshgrid(xf, yf)
+    xf = xf.ravel()
+    yf = yf.ravel()
+
+    return xf * 1e-3, yf * 1e-3
+
+
+def create_sphere(dx=1):
+    x = np.arange(0, 100, dx)
+    y = np.arange(151, 251, dx)
+    x, y = np.meshgrid(x, y)
+    x = x.ravel()
+    y = y.ravel()
+
+    p = ((x - 50)**2 + (y - 200)**2) < 20**2
+    x = x[p]
+    y = y[p]
+
+    # lower sphere a little
+    y = y - 20
+    return x * 1e-3, y * 1e-3
+
+
+def get_density(y):
+    height = 150
+    c_0 = 2 * np.sqrt(2 * 9.81 * height * 1e-3)
+    rho_0 = 1000
+    height_water_clmn = height * 1e-3
+    gamma = 7.
+    _tmp = gamma / (rho_0 * c_0**2)
+
+    rho = np.zeros_like(y)
+    for i in range(len(rho)):
+        p_i = rho_0 * 9.81 * (height_water_clmn - y[i])
+        rho[i] = rho_0 * (1 + p_i * _tmp)**(1. / gamma)
+    return rho
+
+
+def geometry():
```

Comment:
Please use the functions provided already in the pysph.tools.geometry module instead of these.  This plotting function is also not needed here.